### ISSUE TITLE
issue about pretrain

### ISSUE THREAD

**Issue by mbhnwz** [azencot-group/ImagenFew#2]
Hello, I would like to inquire about the issue of whether to set input_channels during pre-training. 
In the paper, DyConv was used to accommodate different channels of various datasets. 
However, in pretain.yaml, input_channels=55 was set, which would lead to data padding:
        if args.input_channels is not None:
            trainset = torch.nn.functional.pad(trainset, (0, args.input_channels - trainset.size(2), 0, 0))
Is there a contradiction here? Is there an error in pretain.yaml?
Looking forward to your reply. Thank you very much.

**Comment by Talgonen** [comment]
You're absolutely right to point this out — this is likely a bug introduced during the repository clean-up. Thanks for catching it!
